.github/workflows/cd.yaml

```
@@ -153,17 +158,25 @@ jobs:
         participant: [company1, company2]
 
     steps:
+      - uses: actions/checkout@v2
+      # Az login is needed to delete the blob terraform state.
+      - name: 'Az CLI login'
+        uses: azure/login@v1
+        with:
+          client-id: ${{ secrets.ARM_CLIENT_ID }}
+          tenant-id: ${{ secrets.ARM_TENANT_ID }}
+          subscription-id: ${{ secrets.ARM_SUBSCRIPTION_ID }}
       - uses: actions/download-artifact@v3
         name: 'Download terraform folder'
         with:
-          name: terraform-${{ matrix.participant }}
-          path: deployment/terraform
+          name: terraform-${{ matrix.participant }}-backend-conf
+          path: deployment/terraform/
```

nit: maybe we can use the full path to make sure we only get backend.conf downloaded here 
```suggestion
          path: deployment/terraform/backend.conf
```